admin/services/pin-generation.service.js

```diff
@@ -17,31 +17,12 @@ const bannedWords = [
   'dim'
 ]
 
-const fourPmToday = () => moment().startOf('day').add(16, 'hours')
-
-const pinExpiryTime = (schoolTimezone = null) => {
-  let toReturn
-  if (config.OverridePinExpiry) {
-    toReturn = moment().endOf('day')
-  } else {
-    toReturn = fourPmToday()
-  }
-  if (schoolTimezone) {
-    // needed to parse the date in the specified timezone and convert to utc for storing
-    toReturn = moment.tz(dateService.formatIso8601WithoutTimezone(toReturn), schoolTimezone).utc()
-  }
-  return toReturn
-}
+const fourPmToday = moment().startOf('day').add(16, 'hours')
```

Comment:
Duplication

Reply:
these consts are required in 2 different services.
1)pin-generation.service for generating school password's `pinExpiresAt`
2)check-start.service for generating check's `pinExpiresAt`

Reply:
and they will also be making an appearance in the school pin generator function :)